src/main/java/net/commuty/parking/model/AccessLog.java

```diff
@@ -32,6 +32,8 @@ public class AccessLog {
 
     private final String reason;
 
+    private final Object attributes;
+
     @JsonCreator
     AccessLog(@JsonProperty("userId") String userId,
```

Comment:
Could you also make this constructor public? The factory methods are too strict regarding the way we create Access Logs now.